src/StreamFactoryInterface.php

```diff
@@ -48,5 +16,5 @@ public function createStreamFromResource($body);
      *
      * @return StreamInterface
      */
-    public function createStreamFromString($body);
+    public function createStream($body = '');
```

Comment:
Probably you didn't get what I mean.

How can you implement the `Zend\Diactoros\Stream` to pass a file in a different mode if you are implementing a Factory of this type?

```php
interface StreamFactoryInterface
{    
    public function createStream($body = '');
}

class DiactorosStreamFactory implements StreamFactoryInterface
{
    public function createStream($body = '')
    {
        return new Stream($body, "?");
    }
}
```

See you cannot make use the second parameter which is useful.

In another note, I understand it is the diactoros constructor and it is a different constructor for `Slim` . In that case I wonder what may be the best way, and will there be a different implementation or constructor type we need ?